workflow/rocoto/gfs_tasks.py

```diff
@@ -659,7 +659,9 @@ def ocnanalprep(self):
         deps = []
         dep_dict = {'type': 'task', 'name': f'{self.cdump}prepoceanobs'}
         deps.append(rocoto.add_dependency(dep_dict))
-        dependencies = rocoto.create_dependency(dep=deps)
+        dep_dict = {'type': 'task', 'name': f'{self.cdump}fcst', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
```

Comment:
I think this should be:
```suggestion
        dep_dict = {'type': 'task', 'name': 'gdasfcst', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
```
When `RUN=gfs`, you will get better backgrounds from the previous `RUN=gdas`.
Also, if you do cycling with `gfs_cyc=1` i.e. run the long forecast at 00z only, there is no `gfsfcst` at 18z and this dependency will never be satisfied.